src/Semantic/Api/Terms.hs

```diff
@@ -87,7 +88,7 @@ data TermOutputFormat
 parseTermBuilder :: (Traversable t, Member Distribute sig, ParseEffects sig m, MonadIO m)
   => TermOutputFormat -> t Blob -> m Builder
 parseTermBuilder TermJSONTree    = distributeFoldMap jsonTerm >=> serialize Format.JSON -- NB: Serialize happens at the top level for these two JSON formats to collect results of multiple blobs.
-parseTermBuilder TermJSONGraph   = termGraph >=> serialize Format.JSONPB
+parseTermBuilder TermJSONGraph   = termGraph >=> serialize Format.JSON
```

Comment:
⚠️ there will be conflicts with #300; hopefully pretty minor, but let me know if you’d like another pair of 👀 on anything.